what_vpn/requests.py

```diff
@@ -18,10 +19,13 @@ def __init__(self, ssl_version=None, **kwargs):
         super().__init__(**kwargs)
 
     def init_poolmanager(self, connections, maxsize, block=False):
+        ssl_context = ssl.SSLContext(self.ssl_version)
```

Comment:
This gives an error if `self.ssl_version == None` (the default).

Reply:
The Signature is [`class ssl.SSLContext(protocol=None)`]( https://docs.python.org/3/library/ssl.html#ssl.SSLContext) so `None` as an argument should be OK